index.js

```diff
@@ -50,8 +51,10 @@ module.exports = {
 function createBot(options) {
   options = options || {};
   options.username = options.username || 'Player';
-  options.version = options.version || false
+  options.version = options.version || false;
   var bot = new Bot();
+  pluginLoader(bot, options);
+  bot.loadPlugins(Object.values(plugins));
```

Comment:
Object.keys(plugins).map(k => plugins[k]) gets the same thing while keeping node6 compatibility